src/examples/forms/input-group/Size.tsx

```diff
@@ -34,7 +36,7 @@ const Example = () => (
         <Label>Plant name generator</Label>
         <InputGroup isCompact>
```

Comment:
I know this is expected, based on how our compact fields are vs. our small buttons, but I can’t help but think this feels wrong. The type sizes between the field and the button are different. :/ 

![image](https://user-images.githubusercontent.com/1230973/118051262-891dc400-b335-11eb-9a93-ce3106237afa.png)

Reply:
Hmm, interesting – now I can't unsee that. This would need to be a component design issue for a separate PR on `react-components` if we decide the change is needed.

Reply:
@nyckie @jon-kerwin what do y’all think here?